### PR TITLE
Correct ApiUmbrellaWeb and fix tests - closes #5

### LINKED ISSUE #5: Fix configurationTests
screencapture-localhost-3000-1447407954592

### PR DESCRIPTION
**Pull request by mauriciovieira**

### CODE REVIEW

**Comment by mauriciovieira** [comment]
@brylie Also, I think that the version should be  _version: "1.0.1"_.

File: tests/server/admin-api/v1/adminUsers.js [hunk 1]
```diff
@@ -14,6 +14,12 @@ Tests for the Admin Users v1 wrapper.
 Tinytest.add(
   "Admin API v1 - Admin Users - getAdmins",
   function (test) {
+    var config = {
+      api_key: Meteor.settings.apiUmbrella.api_key,
```
Comment (brylie) [inline tests/server/admin-api/v1/adminUsers.js]: These have also been changed to camel case: `apiKey`, `authToken`, `baseUrl`.

File: server/apiUmbrellaWeb.js [hunk 2]
```diff
@@ -19,12 +19,12 @@ ApiUmbrellaWeb = function (config) {
   // Get reference to 'this' for methods to use
   var self = this;
 
-  self.baseUrl = config.baseUrl;
+  self.baseUrl = config.base_url;
```
Comment (brylie) [inline server/apiUmbrellaWeb.js]: This should be `config.baseUrl`

File: tests/server/admin-api/v1/apiBackends.js [hunk 3]
```diff
@@ -14,6 +14,12 @@ Tests for the API Backends v1 wrapper.
 Tinytest.add(
   "Admin API v1 - API Backends - getBackends",
   function (test) {
+    var config = {
+      api_key: Meteor.settings.apiUmbrella.api_key,
```
Comment (brylie) [inline tests/server/admin-api/v1/apiBackends.js]: Again, using camel case now, to follow common JS practice. I.e. `apiKey`, `authToken`, `baseUrl`.

**Pull request merged**